source/Coop/Mod/Logging.cs

```diff
@@ -26,12 +26,12 @@ public static void Init(Target[] appSpecificTargets)
                 IncludeSourceInfo = true
             };
 
-            config.AddRule(LogLevel.Info, LogLevel.Fatal, logFile);
+            config.AddRule(LogLevel.Trace, LogLevel.Fatal, logFile);
```

Comment:
I'd prefer to leave that on Info. There's a lot of messages on Trace once you start playing.